.github/workflows/test-c3.yml

```diff
@@ -81,6 +81,36 @@ jobs:
           CLOUDFLARE_API_TOKEN: ${{ secrets.C3_TEST_CLOUDFLARE_API_TOKEN }}
           CLOUDFLARE_ACCOUNT_ID: ${{ secrets.C3_TEST_CLOUDFLARE_ACCOUNT_ID }}
 
+  e2e-quarantine:
+    needs: cleanup
+    name: "E2E (Quarantine)"
+    continue-on-error: true
+    if: |
+      github.event.pull_request.user.login != 'dependabot[bot]'
```

Comment:
Why do we not want to test on dependabot PRs?

Reply:
E2E on dependabot PRs is handled in `e2e-only-dependabot-bumped-framework`. I do think I need to update that to have a similar job for quarantined frameworks though